master.java

```diff
@@ -597,6 +637,37 @@ public static void intro()
     System.out.println();
 }
 
+/*********************************************************/
+/*********************************************************/  
+  
+/*
+ * userBased
+ * param: int points, List<String> acceptedWords
+ * returns: String
+ * This function is used for the player to create their own baseword.
+ */
```

Comment:
If this function is not being used please remove it. Do whatever needs to be done to resolve this.